### ISSUE TITLE
Merging multiple documents together into 1 large document.

### ISSUE THREAD

**Issue by JMurphyWeb** [karmaradio/karma#349]
Initially we are looking into getting this flow testable for only merging one document (contract) after accepting an offer.  As mentioned on our call today, 1 document might be desirable for docusign reasons so need to think of a way to get all relevant documents merged into 1 before merging data into it.

1st iteration:
PM uploads all documents in 1 go. Meaning any conditional forms will either always be there or never.

2nd iteration:
Look into merging necessary documents (including conditionals depending on offer/startpack) before merging the data into it.

**Comment by JMurphyWeb** [comment]
@jonscottfilm could you add your priority to this please?

Adding to Sprint 3 to be worked on if there is time and if it is a priority.

**Comment by jonscottfilm** [comment]
If we're not Docusigning in Sp3, then this can take a back seat.
BUT, if what we implement here, will inform our Docusign process, then we need to think on it now.

Presuming the PM uploads individual documents (function for which is currently in place), the immediate solution would be WE merge all _relevant_ (i.e. any conditional) documents, into one combined document, to then go off to Docusign.

**Comment by jonscottfilm** [comment]
We can't release to test project, until merge>docusign approval chain is in place, which I imagine, is not until at least end Sp4.

So, if thinking about combining multiple documents in Sp3 is a hindrance, then let's just focus on merging 1 document successfully.

**Comment by JMurphyWeb** [comment]
Got it and I agree. Worth putting a priority label on it so it doesn't get lost in the backlog..

**Comment by jonscottfilm** [comment]
Roger.

I'm gonna edit the title to 'Handling merge of multiple documents'.
